src/app/app.tsx

```diff
@@ -79,6 +96,7 @@ export function App() {
               if (evt.key === 'Enter') {
                 const target = evt.target as HTMLInputElement;
                 setNpmPackageName(target.value?.toLocaleLowerCase());
+                setSelectedVersion('versions');
```

Comment:
Could we use `null` to represent the root state? That would be preferable for shared urls as well.